CONTRIBUTING.md

```diff
@@ -130,3 +130,15 @@ In this repo, Github Actions are used for:
 
 - delete merged PR branches
 - automatically create draft PRs for pushed branches
+
+### Contributing Process
+
+This is a step-by-step guide to how you are able to contribute to Vaken. After you find an issue you would like to work on:
+
+1. `git clone <link to the repository>` into your desired directory

```

Comment:
I would recommend directly inlining the public link to vaken here (https://github.com/VandyHacks/vaken) as it is inside the vaken repo.